content/en/docs/Getting Started/quickstart-kind.md

```diff
@@ -0,0 +1,168 @@
+
+<pre>
+Kubernetes control plane is running at https://127.0.0.1:39509
+CoreDNS is running at
+https://127.0.0.1:39509/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+</pre>
+
+## Install Tekton Pipelines
+
+1. To install the latest version of Tekton Pipelines, use `kubectl`:
+
+   ```bash
+   kubectl apply --context kind-tektoncd --filename \
+   https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+   ```
+
+1. Monitor the installation:
+
+   ```bash
+   kubectl get pods --context kind-tektoncd --namespace tekton-pipelines --watch
+   ```
+
+When all components show `Running` under the `STATUS` column the installation
+is complete.
+
+Hit *Ctrl + C* to stop monitoring.
+
+## Create and run a basic Task
+
+A **Taks** defines a series of **steps** that run sequentially to perform some
+build work. Every Task runs as a pod on your Kubernetes cluster, with each step
+running in its own container.
+
+1. To create a task, open your favorite editor and create a file named
+   `hello-world.yaml` with the following content:
+
+	 ```yaml
+	 apiVersion: tekton.dev/v1beta1
+	 kind: Task
+	 metadata:
+	 	name: echo-hello-world
+	 spec:
+	 	steps:
+	 		- name: echo
+	 			image: alpine
+	 			command:
+	 				- echo
+	 			args:
+	 				- "Hello World"
+	 ```
+
+1. Apply the Task to your cluster:
+
+   ```bash
+   kubectl apply --context kind-tektoncd --filename hello-world.yaml
+   ```
+
+1. To run this task, you must instantiate it using `TaskRun`. Create another
+   file named `hello-world-run.yaml` with the following content:
+
+   ```yaml
+   apiVersion: tekton.dev/v1beta1
+   kind: TaskRun
+   metadata:
+     name: echo-hello-world-task-run
+   spec:
+     taskRef:
+       name: echo-hello-world
+   ```
+
+1. Apply the TaskRun to your cluster to launch the task:
+
+   ```bash
+   kubectl apply --context kind-tektoncd --filename hello-world-run.yaml
+   ``` 
+
+1. Verify that everything worked correctly:
+
+   ```bash
+   kubectl describe pod echo-hello-world-task-run
+   ```
+
+     The output of this command includes a lot of information about the pod.
+
+     <pre>
+     Name:         echo-hello-world-task-run-pod-c67wv
+     Namespace:    default
+     Priority:     0
+     Node:         tektoncd-control-plane/172.19.0.3
+     Start Time:   Thu, 23 Sep 2021 13:49:40 -0400
+     Labels:       app.kubernetes.io/managed-by=tekton-pipelines
+                   tekton.dev/task=echo-hello-world
+                   tekton.dev/taskRun=echo-hello-world-task-run
+     Annotations:  pipeline.tekton.dev/release: adaae90
+                   tekton.dev/ready: READY
+     Status:       Succeeded
+     IP:           10.244.0.7
+     IPs:
+       IP:           10.244.0.7
+     Controlled By:  TaskRun/echo-hello-world-task-run
+     ...
+     </pre>
+
+     The line `Status: Succeeded` confirms that TaskRun completed with no errors.
```

Comment:
instead of showing them the pod, it might be a bit nicer to show them the taskrun, e.g. `kubectl --context kind-tektoncd get taskrun echo-hello-world-task-run` would show something simple like:

```
NAME                               SUCCEEDED    REASON       STARTTIME   COMPLETIONTIME
echo-hello-world-task-run          True         Succeeded    22h         22h
```

plus the pods are (kind of) an implementation detail of tekton anyway 🤞  (tho in reality is not like ppl can really ignore them...)

Reply:
Updated